src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AutomaticEndAppealForNonPaymentEaHuTriggerTest.java

```diff
@@ -128,4 +128,206 @@ void handling_should_throw_if_null_callback() {
             .isExactlyInstanceOf(NullPointerException.class);
     }
 
+    private void dataSetUp() {
```

Comment:
After you set the DetentionTo Yes, the class name is not apt for these tests any more. By default, detention should be set to No and then in more special scenarios, you set them explicitly.

Reply:
Done.